src/polyfills/datalist/datalist-fr.hbs

```diff
@@ -20,7 +20,7 @@
 	<form action="#" method="get">
 		<div class="form-group">
 			<label for="city">Ville</label>
-			<input class="form-control" type="text" id="city" name="city" list="suggestions">
+			<input class="form-control" type="text" autocomplete="address-level2" id="city" name="city" list="suggestions">
```

Comment:
In order to use "address-line1", "address-line2" or "address-line3", you must be using "street-address" also.